travis/set_tags.sh

```diff
@@ -0,0 +1,13 @@
+if [ -z "$TRAVIS_TAG" ]; then
```

Comment:
I think we want to deploy to a tag:

`#bin-commitHash-os-cpu`, where `commitHash` could either be the commit hash, or `master` perhaps.